requirements-minimal.txt

```diff
@@ -9,3 +9,4 @@ pynwb>=1.4.0
 psutil>=5.8.0
 tqdm>=4.60.0
 dandi>=0.46.2
+pandas
```

Comment:
Also don't think we need it here at all either - it should be included with PyNWB

Reply:
If we import a library, we should list it as a requirement, even if it is a requirement of another required repo. What if PyNWB removes pandas as a requirement? Our code would then break. Also, we may have certain version requirements for features of pandas that PyNWB does not have